format/spec.md

```diff
@@ -702,6 +703,49 @@ Blob metadata is a struct with the following fields:
 | _optional_ | _optional_ | **`properties`** | `map<string, string>` | Additional properties associated with the statistic. Subset of Blob properties in the Puffin file. |
 
 
+#### Partition statistics
+
+Partition statistics files are based on [Partition Statistics file spec](#partition-statistics-file). 
+Partition statistics are not required for reading or planning and readers may ignore them.
+Each table snapshot may be associated with at most one partition statistic file.
+A writer can optionally write the partition statistics file during each write operation. If the statistics file is written for the specific snapshot,
```

Comment:
Nit: I am not too sure these two sentences add much value, it is the case for any file reference in Iceberg , isnt it?

  
```A writer can optionally write the partition statistics file during each write operation. If the statistics file is written for the specific snapshot, it must be registered in the table metadata file to be considered as a valid statistics file for the reader.```

Reply:
I have shortened it a bit. Even though it seems implicit, It links back to how it is tracked and when it is valid. I remember getting some comment to add this statement.